FOR_ANALYSTS.md

```diff
@@ -135,6 +135,23 @@ data = quandl.get_table('ZACKS/FC', paginate=True, ticker=['AAPL', 'MSFT'], per_
 
 In this query we are asking for more pages of data, `ticker` values of either `AAPL` or `MSFT` and a `per_end_date` that is greater than or equal to `2015-01-01`. We are also filtering the returned columns on `ticker`, `per_end_date` and `comp_name` rather than all available columns. The output format is `pandas`.
 
+Download table data as a zip file. You can download all the table data in a data table in a single call. The following will download the entire F1 table data as a zip file to your current working directory:  
+
+```python
+import quandl
+data = quandl.bulkdownloadtable('MER/F1')
+```
+
+You can also specify where to download the zip file:  
+
+```python
+import quandl
+data = quandl.bulkdownloadtable('MER/F1', filename='/my/path/MER_F1_DB.zip')
+```
+After the download is finished, the filename of the downloaded zip file will be returned.
+
+Sometime it takes a while to generate the zip file, you'll get a message while the file is being generated. Once the file is generated, it will start download the zip file.
```

Comment:
Small grammar issues- should say `Sometimes` instead of `Sometime`. Also `it will start download the` should be `it will start the download of the`

Reply:
haha, will update!